android/guava/src/com/google/common/io/BaseEncoding.java

```diff
@@ -627,6 +627,7 @@ public void close() throws IOException {
               }
             }
           }
+          bitBufferLength = 0;
```

Comment:
I'd move this just after the increment to writtenChars (so, just in case the write of the padding character fails and someone tries to re-close the stream anyway, we get it correctly)

Reply:
Done!